source/core/src/core/silver/domain/transformations/migrations_transformation.py

```diff
@@ -0,0 +1,64 @@
+import pyspark.sql.functions as F
+from pyspark.sql import Column, DataFrame, SparkSession
+
+import core.utility.datetime_helper as datetime_helper
+from core.bronze.domain.constants.column_names.bronze_migrated_transactions_column_names import (
+    BronzeMigratedTransactionsColumnNames,
+    BronzeMigratedTransactionsValuesFieldNames,
+)
+from core.silver.domain.constants.column_names.silver_measurements_column_names import SilverMeasurementsColumnNames
+from core.silver.domain.constants.enums.read_reason_enum import ReadReasonEnum
+from core.silver.domain.constants.enums.status_enum import StatusEnum
+
+
+def create_by_migrated_transactions(spark: SparkSession, migrated_transactions: DataFrame) -> DataFrame:
```

Comment:
How about this? Same naming used for submitted 
```suggestion
def transform(spark: SparkSession, migrated_transactions: DataFrame) -> DataFrame:
```

Reply:
When I started the PR submitted was named "create_by_submitted_transactions"! I agree :D